src/generator/AutoRest.Python.Azure/TransformerPya.cs

```diff
@@ -239,5 +235,28 @@ public static void CorrectFilterParameters(CodeModel serviceClient)
                 }
             }
         }
+
+        /// <summary>
+        /// Creates azure specific properties.
+        /// </summary>
+        /// <param name="codeModelient"></param>
+        private static void AddAzureProperties(CodeModel codeModel)
+        {
+            var acceptLanguage = codeModel.Properties
+                .FirstOrDefault(p => AzureExtensions.AcceptLanguage.EqualsIgnoreCase(p.SerializedName));
+
+            AzureExtensions.AddAzureProperties(codeModel);
+            codeModel.Remove(codeModel.Properties.FirstOrDefault(p => p.Name == "long_running_operation_retry_timeout"));
+            codeModel.Remove(codeModel.Properties.FirstOrDefault(p => p.Name == "generate_client_request_id"));
+            codeModel.Remove(codeModel.Properties.FirstOrDefault(p => p.Name == "accept_language"));
+
+            if (acceptLanguage != null) // && acceptLanguage.DefaultValue != "en-US"
```

Comment:
It might be worth investigating if there's any merit in the second half of this if statement that was commented out. Additionally if would be worth seeing if any of the SDK specs actually have an explicit accept language so that we can exercise this code.